added simple travis-ci to qdr-shipshape repo

nicob87 opened the pull request.

fgiorgetti commented on test/travis/send_receive/setup.go:
```diff
@@ -0,0 +1,100 @@
+		Data: map[string]string{
+			"small-message.txt": messaging.GenerateMessageContent("ThisIsARepeatableMessage", 1024),
+		},
+	})
```
@nicob87 maybe use the "GenerateSmallMediumLargeMessagesConfigMap" which is available in the pkg instead?

nicob87 replied:
done.

fgiorgetti reviewed (commented):
@nicob87 I like the idea of running E2E test on the stuff we have in the pkg/.
I would say we could also add a simple test to validate the "pkg/clients/python" as well.

Commit:
Added simple travis-ci to qdr-shipshape repo. The idea is to have in CI a simple
exercise of all features added to the qdr-shipshape library. And also unit-tests.
